app/controllers/va_facilities_controller.rb

```diff
@@ -43,7 +43,7 @@ def show
       marker.shadow nil
       marker.json({ id: facility.id })
     end
-    created_practices = Practice.get_facility_created_practices(station_number, nil, 'a_to_z', nil)
+    created_practices = Practice.sort_by_retired.get_facility_created_practices(station_number, nil, 'a_to_z', nil)
```

Comment:
since we already sort this I don't think we need `.sort_by_retired` here or on line 67